### PR TITLE
Updated Sendai

### PR DESCRIPTION
**Pull request by mharricharan**
Sendai Architecture is amazing

### CODE REVIEW

File: web_source/cities/asia/japan/sendai.md [hunk 1]
```diff
@@ -4,12 +4,13 @@ Country: Japan (Asia)
...
 ##### Also known as:
 
-FIXME
+Sendai-shi
 
 ### What we would like to see there
 
-FIXME
+Architecture
+![Architecture](https://upload.wikimedia.org/wikipedia/commons/thumb/f/ff/Sendai_Zuiho-den_Mausoleum_4.jpg/1280px-Sendai_Zuiho-den_Mausoleum_4.jpg)
```
Comment (AngharadGreen) [inline web_source/cities/asia/japan/sendai.md]: Can you give more information of what this image is of?

**Review by AngharadGreen — CHANGES_REQUESTED** [review]
Thank you for adding in your changes, it looks great, I have left one comment for you to review.

**Comment by mharricharan** [comment]
Thanks @AngharadGreen Changes made.

**Review by AngharadGreen — APPROVED** [review]
Looks great

**Pull request merged**